spec/operators/windowTime-spec.js

```diff
@@ -123,6 +131,37 @@ describe('Observable.prototype.windowTime', function () {
     expectObservable(result, unsub).toBe(expected, values);
   });
 
+  it('should dispose window Subjects if the outer is unsubscribed early', function (done) {
+    var virtualTimeScheduler = new Rx.VirtualTimeScheduler();
+    var source = new Rx.Subject();
+    var result = source.windowTime(1000, 1000, virtualTimeScheduler);
+    var window;
+    var subscription;
+
+    virtualTimeScheduler.schedule(function () {
+      subscription = result.subscribe(function (w) { window = w; });
+    }, 0);
+    virtualTimeScheduler.schedule(function () { source.next('a'); }, 0);
+    virtualTimeScheduler.schedule(function () { source.next('b'); }, 10);
+    virtualTimeScheduler.schedule(function () { source.next('c'); }, 20);
+    virtualTimeScheduler.schedule(function () { subscription.unsubscribe(); }, 30);
+    virtualTimeScheduler.schedule(function () { source.next('d'); }, 40);
+    virtualTimeScheduler.schedule(function () { source.next('e'); }, 50);
+    virtualTimeScheduler.schedule(function () { source.next('f'); }, 60);
+    virtualTimeScheduler.schedule(function () { source.next('g'); }, 70);
+    virtualTimeScheduler.schedule(function () {
+      try {
+        window.subscribe();
+      }
+      catch (err) {
+        expect(err.message).toBe('Cannot subscribe to a disposed Subject.');
+        done();
```

Comment:
should this test be async test while using virtualtimescheduler? locally tried sync assertion as other PR (https://github.com/ReactiveX/RxJS/pull/1012/files#diff-f0b7f1f59929d2619cbf53504f594616R219) seems works.